app/src/main/java/com/android/sample/MainActivity.kt

```diff
@@ -134,5 +135,10 @@ fun ActivitiesApp(uid: String, startDestination: String) {
       }
       composable(Screen.EDIT_PROFILE) { EditProfileScreen(profileViewModel, navigationActions) }
     }
+    navigation(startDestination = Screen.LIKED_ACTIVITIES, route = Route.LIKED_ACTIVITIES) {
+      composable(Screen.LIKED_ACTIVITIES) {
+        LikedActivitiesScreen(listActivitiesViewModel, navigationActions, profileViewModel)
+      }
+    }
```

Comment:
The addition of the LikedActivities screen to the navigation graph in MainActivity is well executed. Integrating this feature directly into the main navigation flow ensures that users can easily access their liked activities from anywhere within the app.